src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr

```diff
@@ -1,25 +1,20 @@
 use vs
 
 package name=PortableFacades
-        version=1.0.0.0
+        version=1.1.0.0
```

Comment:
Do we need to make corresponding version changes anywhere in VS as part of insertion?